sdk/github.md

```diff
@@ -205,8 +205,28 @@ tied in with the [SDK Upload Action](https://github.com/ably/sdk-upload-action).
 
 ## Labels
 
-We are slowly converging on a set of labels that are useful for our context and these are being duplicated across client libraries when needed.
-This list is canonically defined [here](https://github.com/ably/ably-common/blob/main/github/labels.yml).
+The following table canonically defines labels we use in common across our open source SDK repositories:
+
+| Name | Color | Description | Additional Notes |
+| ---- | ----------- | ----- | ---------------- |
+| `blocked-by-ably` | ![fbca04](https://img.shields.io/badge/-fbca04-fbca04) | We can't proceed until something under our direct control, in a different codebase, happens. | Comments should be added to indicate what the blockage is (e.g. another SDK repository). |
+| `blocked-by-external` | ![fbca04](https://img.shields.io/badge/-fbca04-fbca04) | We can't proceed until something outside of Ably's direct control happens. | Comments should be added to indicate what the blockage is. |
+| `bug` | ![d73a4a](https://img.shields.io/badge/-d73a4a-d73a4a) | Something isn't working. It's clear that this does need to be fixed. | Usually implies that related changes can be released in a `patch` version bump. |
+| `breaking` | ![ec4b42](https://img.shields.io/badge/-ec4b42-ec4b42) | Affects the developer experience when working in our codebase. | Implies a need to release related changes in a `major` version bump. |
+| `code-quality` | ![ff88cc](https://img.shields.io/badge/-ff88cc-ff88cc) | Affects the developer experience when working in our codebase. |
+| `documentation` | ![0075ca](https://img.shields.io/badge/-0075ca-0075ca) | Improvements or additions to public interface documentation (API reference or readme). |
+| `enhancement` | ![a2eeef](https://img.shields.io/badge/-a2eeef-a2eeef) | New feature or request. | Implies a need to release related changes in a `minor` version bump. |
```

Comment:
I am not sure if that should be the case, but what if we do a work that improves some internal working of a visible function (eg - speed improvement)? Isn't that also an enhancement? And is it a breaking change?

Reply:
What you describe doesn't sound like a breaking change, but could be an `enhancement` depending on how impactful it is and whether we feel customers need to know about it. I'm not sure what could be added to this table to enhance that understanding though, so open to suggested changes or additions to the wording here. 🤔

Reply:
I would personally prefer a different label for a new feature (such as new-feature) and separate it from enhancement. But if we have to keep them on the same label, I think description might be  something like "New feature or functionality enhancement" and then the implication for the release would only be for cases when new features are added.

Reply:
I will resolve this conversation and leave it to you whether you want to change anything here as we have agreed on this in the office

Reply:
I've taken aboard your suggestion, with slightly different wording: https://github.com/ably/engineering/pull/44/commits/34456b2a587f61429ba0fab21e181761ab79bee9